frontend/src/concepts/dashboard/DashboardPopupIconButton.tsx

```diff
@@ -3,16 +3,27 @@ import { Button, ButtonProps, Icon } from '@patternfly/react-core';
 
 type DashboardPopupIconButtonProps = Omit<ButtonProps, 'variant' | 'isInline'> & {
   icon: React.ReactNode;
+  iconStatus?: 'custom' | 'info' | 'success' | 'warning' | 'danger';
```

Comment:
use IconComponentProps['status']

Reply:
better, just use iconProps: Omit<IconComponentProps, 'isInline'> and then spread it across the icon